webcompat/webhooks/index.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
```

Comment:
@marimeireles Is it a mistake in the commit?

Reply:
It is. Didn't see it. Thanks.